src/main/java/com/hubspot/jackson/datatype/protobuf/builtin/serializers/DurationSerializer.java

```diff
@@ -1,17 +1,22 @@
 package com.hubspot.jackson.datatype.protobuf.builtin.serializers;
 
-import java.io.IOException;
-
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.databind.SerializerProvider;
 import com.google.protobuf.Duration;
 import com.google.protobuf.util.Durations;
+import com.hubspot.jackson.datatype.protobuf.ProtobufJacksonConfig;
 import com.hubspot.jackson.datatype.protobuf.ProtobufSerializer;
+import java.io.IOException;
 
 public class DurationSerializer extends ProtobufSerializer<Duration> {
 
+  @Deprecated
```

Comment:
Should we add javadoc for these deprecations?

Reply:
👍 4197fefc1660730bd3090af6582c978a96f86b69